docs/resources/aviatrix_controller_access_allow_list_config.md

```diff
@@ -0,0 +1,48 @@
+---
+subcategory: "Settings"
+layout: "aviatrix"
+page_title: "Aviatrix: aviatrix_controller_access_allow_list_config"
+description: |-
+  Creates and manages an Aviatrix Controller's Access Allow List  Config
+---
+
+# aviatrix_controller_access_allow_list_config
+
+!> **WARNING:** If any of the Allow List IPs are incorrect, the Controller will be inaccessible.
+
+The **aviatrix_controller_access_allow_list_config** resource allows management of an Aviatrix Controller's Access Allow List.
```

Comment:
... enables configuration of a set of IP addresses allowed HTTP(s) access to an Aviatrix Controller.

Please confirm if that is correct